integration-cli/docker_cli_daemon_test.go

```diff
@@ -152,6 +153,79 @@ func (s *DockerDaemonSuite) TestDaemonStartIptablesFalse(c *check.C) {
 	}
 }
 
+// Make sure we cannot shrink base device at daemon restart.
+func (s *DockerDaemonSuite) TestDaemonRestartWithInvalidBasesize(c *check.C) {
+	s.d.storageDriver = "devicemapper"
+	if err := s.d.Restart(); err != nil {
```

Comment:
`c.Assert(s.d.Start(), check.IsNil)`
Also you're using restart and _start_ in the comment below but I guess you meant Start()